docs/en/modules/ws2812.md

```diff
@@ -23,7 +23,15 @@ Initialize UART0 (TXD0) too if `ws2812.MODE_DUAL` is set.
 `ws2812.init([mode])`
 
 #### Parameters
-- `mode` (optional) either `ws2812.MODE_SINGLE` (default if omitted) or `ws2812.MODE_DUAL`
+- `mode` (optional) either `ws2812.MODE_SINGLE` (default if omitted), `ws2812.MODE_SINGLE_OPEN`, or `ws2812.MODE_DUAL`
```

Comment:
I was not sure where ws2812 currently configures the second pin; there isn't the code I expected in `ws2812_init`, so I left it be.  Yes, in principle, I think there could be a `MODE_DUAL_OPEN`.